zio-http/src/test/scala/zhttp/service/ServerSpec.scala

```diff
@@ -204,10 +208,21 @@ object ServerSpec extends HttpRunnableSpec(8088) {
       }
   }
 
-  override def spec = {
+  override def spec =
     suiteM("Server") {
-      app.as(List(staticAppSpec, dynamicAppSpec, responseSpec, requestSpec)).useNow
+      app.as(List(serverStartSpec, staticAppSpec, dynamicAppSpec, responseSpec, requestSpec)).useNow
     }.provideCustomLayerShared(env) @@ timeout(30 seconds)
+
+  def serverStartSpec = suite("ServerStartSpec") {
+    testM("make") {
+      (Server.port(0) ++ Server.app(Http.ok)).make.use { start =>
+        val actual = Client
+          .request(s"http://localhost:${start.port}")
+          .map(_.status)
+
+        assertM(actual)(equalTo(Status.OK))
```

Comment:
- [ ] Test should assert if the port is set. 
- [ ] Test should assert that if no port is provided, a random port is assigned and returned.